electron.js

```diff
@@ -1,39 +1,39 @@
-'use strict'
-const electron = require('electron')
+'use strict';
```

Comment:
Not 100% sure what this file is, and if we should be modifying it or if it should be part of the .eslintignore file so it doesn't get linted/formatted.